test/plugins/auctionHouseModule/createBid.test.ts

```diff
@@ -0,0 +1,158 @@
+import test, { Test } from 'tape';
+import spok, { Specifications } from 'spok';
+import { sol, token } from '@/types';
+import {
+  metaplex,
+  killStuckProcess,
+  createNft,
+  spokSamePubkey,
+  spokSameAmount,
+} from '../../helpers';
+import { createAuctionHouse } from './helpers';
+import { findAssociatedTokenAccountPda } from '@/plugins';
+import { Bid } from '@/plugins/auctionHouseModule/Bid';
+import { AccountNotFoundError } from '@/errors';
+
+killStuckProcess();
+
+test('[auctionHouseModule] create a new bid on an Auction House', async (t: Test) => {
+  // Given we have an Auction House and an NFT.
+  const mx = await metaplex();
+  const nft = await createNft(mx);
+  const { auctionHouse, client } = await createAuctionHouse(mx);
+
+  // When we list that NFT for 6.5 SOL.
+  await client
+    .list({
+      mintAccount: nft.mintAddress,
+      price: sol(6.5),
+    })
+    .run();
```

Comment:
You shouldn't need to create a listing in order to test creating a bid.

I suspect this was a work-around for the misuse of the `tokenAccount` input parameter.

Reply:
Yeah, that was it

Reply:
But I actually didn't realize I misused the token account.